assets/css/style.css

```diff
@@ -429,9 +429,6 @@ nav ul li a:focus {
         padding-top: 20px;
         padding-bottom: 20px;
     }
-    #footer-video-container {
```

Comment:
This page is now kind of messed up on smaller screens.

<img width="1025" alt="screen shot 2017-02-07 at 11 45 33" src="https://cloud.githubusercontent.com/assets/109850/22688088/fcad80a2-ed2a-11e6-9dc2-5307fd1c0656.png">